src/MSBuild.Abstractions/BaselineProject.cs

```diff
@@ -60,8 +60,9 @@ private static string GetCurrentTFM(ImmutableArray<string> globalProperties, Unc
                     $"{MSBuildFacts.TargetFrameworkNodeName} is not set in {nameof(project.FirstConfiguredProject)}");
             }
 
-            // This is pretty much never gonna happen, but it was cheap to write the code
-            return MSBuildHelpers.IsNotNetFramework(rawTFM) ? StripDecimals(rawTFM) : rawTFM;
+            // MSBuildHelpers.IsNotNetFramework is pretty much never gonna happen...
```

Comment:
I would change this to just say `MSBuildHelpers.IsWindows(rawTFM) can happen when coverting a UWP`